pegasus/sites.v3/code.org/public/professional-learning/self-paced.haml

```diff
@@ -20,7 +20,7 @@ theme: responsive_full_width
         =hoc_s(:self_paced_pl_hero_desc)
     %img{src: "../shared/images/banners/self-paced-pl-hero.png", alt: ""}
 
-%section
+%section.bg-neutral-light
```

Comment:
Removing the banner created back-to-back white sections. Added this here to fix that.

| Before | After |
| ---- | ---- |
| ![code org_professional-learning_self-paced(Code org - 1366x768)](https://github.com/code-dot-org/code-dot-org/assets/9256643/51af6b05-ef5c-4933-9134-391b6b6e6bac) | ![localhost code org_3000_professional-learning_self-paced(Code org - 1366x768)](https://github.com/code-dot-org/code-dot-org/assets/9256643/10abbcfb-84e6-43ae-9f0e-d02bdcf7945b) |